rust/lance-arrow/Cargo.toml

```diff
@@ -22,6 +22,7 @@ arrow-schema = { workspace = true }
 arrow-select = { workspace = true }
 bytes = { workspace = true }
 half = { workspace = true }
+jsonb = "0.5.3"
```

Comment:
Minor nit: Can we put this in the workspace cargo, even if it is only used in one place?  I find it is just simpler to have all versions in one spot.

Reply:
Fixed.